Use python2 as Build-Depend
Author: Han Gao <gaohan@iscas.ac.cn>

Last-Update: 2022-11-15

--- recode-3.6.1.orig/doc/Makemore
+++ recode-3.6.1/doc/Makemore
@@ -52,7 +52,7 @@ MNEMONICS_DS = keld-email/iso10646.def k
 NOMS_CARACS = noms_juc_19981216.txt
 NOMS_CARACS = NomsSeulsfinal.lst
 
-PYTHON = python
+PYTHON = python2
 SURF = netscape
 WGET = wget
 
--- recode-3.6.1.orig/doc/tables.py
+++ recode-3.6.1/doc/tables.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
 #                                                    -*- coding: latin-1 -*-
 # Automatically derive `recode' table files from various sources.
 # Copyright © 1993, 1994, 1997, 1998, 1999, 2000 Free Software Foundation, Inc.
@@ -21,7 +21,7 @@
 """\
 `tables.py' derives `recode' table files from various sources.
 
-Usage: python tables.py [OPTION]... DATA-FILE...
+Usage: python2 tables.py [OPTION]... DATA-FILE...
 
   -e  produce C source file for explode data (explode.c)
   -l  produce C source file for libiconv charsets (libiconv.h)
--- recode-3.6.1.orig/src/Makefile.am
+++ recode-3.6.1/src/Makefile.am
@@ -62,7 +62,7 @@ TAGS_DEPENDENCIES = $(L_STEPS)
 
 LEX = @LEX@
 PERL = perl
-PYTHON = python
+PYTHON = python2
 
 $(srcdir)/decsteps.h: stamp-steps
 $(srcdir)/inisteps.h: stamp-steps
--- recode-3.6.1.orig/src/Makefile.in
+++ recode-3.6.1/src/Makefile.in
@@ -145,7 +145,7 @@ TAGS_DEPENDENCIES = $(L_STEPS)
 
 LEX = @LEX@
 PERL = perl
-PYTHON = python
+PYTHON = python2
 mkinstalldirs = $(SHELL) $(top_srcdir)/mkinstalldirs
 CONFIG_HEADER = ../config.h
 CONFIG_CLEAN_FILES = 
--- recode-3.6.1.orig/tests/Makefile.am
+++ recode-3.6.1/tests/Makefile.am
@@ -25,7 +25,7 @@ names.m4 quoted.m4 testdump.m4 utf7.m4 u
 EXTRA_DIST = atgeneral.m4 suite.m4 testsuite $(SUITE) \
 bigauto.py dumps-data quoted-data utf7-data
 
-PYTHON = python
+PYTHON = python2
 
 all-local: atconfig testsuite
 
--- recode-3.6.1.orig/tests/Makefile.in
+++ recode-3.6.1/tests/Makefile.in
@@ -112,7 +112,7 @@ SUITE = african.m4 base64.m4 combine.m4
 EXTRA_DIST = atgeneral.m4 suite.m4 testsuite $(SUITE) bigauto.py dumps-data quoted-data utf7-data
 
 
-PYTHON = python
+PYTHON = python2
 mkinstalldirs = $(SHELL) $(top_srcdir)/mkinstalldirs
 CONFIG_HEADER = ../config.h
 CONFIG_CLEAN_FILES =  atconfig
--- recode-3.6.1.orig/tests/bigauto.py
+++ recode-3.6.1/tests/bigauto.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 # Copyright © 1997, 1999, 2000 Progiciels Bourbeau-Pinard inc.
 # François Pinard <pinard@iro.umontreal.ca>, 1997.
 
